include/mbgl/ios/MGLCalloutViewProtocol.h

```diff
@@ -0,0 +1,29 @@
+#import <Foundation/Foundation.h>
+
+@protocol MGLCalloutViewDelegate;
+
+@protocol MGLCalloutViewProtocol <NSObject>
+
+@property (nonatomic, copy) NSString *title, *subtitle;
```

Comment:
Please audit this header for nullability and use the `NS_ASSUME_NONNULL_BEGIN` and `NS_ASSUME_NONNULL_END` macros to tell the compiler you’ve done so. For Xcode 6.4 compatibility, you’ll need to `#import "MGLTypes.h"`.